examples/jsm/controls/TrackballControls.js

```diff
@@ -36,6 +38,8 @@ class TrackballControls extends EventDispatcher {
 		this.zoomSpeed = 1.2;
 		this.panSpeed = 0.3;
 
+		this.rotateThreshold = 0.5;
```

Comment:
I'm adding the documentation right now. Can you please explain the purpose of this new property?

Reply:
It also seems this new property affects a previous code path:

https://github.com/mrdoob/three.js/blob/8d2fb0830d4446ff97fefe178446795c0ad01d31/examples/jsm/controls/TrackballControls.js#L184-L187

Why was this change required?

Reply:
If I remember correctly, the angle between the 2 fingers sometimes returns the opposite angle depending on the order the touch events are provided to the `atan2` function. This resulted in the rotation jittering in between the current rotation, and the 180º rotation. The `rotateThreshold` property controls above which rotation angle the rotation is ignored in order to prevent this jittering.

I'll look tonight if the second if is necessary.

Reply:
> The rotateThreshold property controls above which rotation angle the rotation is ignored in order to prevent this jittering.

I would be also good to make the property internal if possible. I doubt users can properly configure it since it's purpose is not obvious.